environment

```diff
@@ -107,6 +107,9 @@ GOOGLE_APPLICATION_CREDENTIALS_SECRETS_NAME="gcp-credentials.json"
 GOOGLE_APPLICATION_SECRETS_SECRETS_NAME="application_secrets.json"
 ES_ALLOWED_SOURCE_IP_SECRETS_NAME="es_source_ip"
 DSS_DEBUG=0
+TF_VAR_PROJECT=dcp
+TF_VAR_SERVICE=dss
```

Comment:
Are the `TF_VAR_SERVICE`, etc. names used?  I only see a ref to `var.SERVICE`, etc.

Reply:
they are, terraform requires the env variables to be defined as `TF_VAR_{x}` and they get overridden when TF is parsing the values for the variables that are marked with the same `x` value. This is why the variables are defined, but dont have defaults attached to them.